packages/flutter_hooks/test/use_listenable_map_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/src/framework.dart';
+import 'package:flutter_hooks/src/hooks.dart';
+import 'mock.dart';
+
+void main() {
+  testWidgets('debugFillProperties', (tester) async {
+    await tester.pumpWidget(HookBuilder(
+      builder: (context) {
+        final listenable = ValueNotifier<int>(42);
+        useListenableMap<ValueNotifier<int>, bool Function(), bool>(
+            listenable, () => listenable.value.isOdd);
+        return const SizedBox();
+      },
+    ));
```

Comment:
Nit: add trailing comma and format

Reply:
I have too much knowledge to learn, so I will study. Thank you, Remi 👍